backend/dataall/modules/datasets/api/dataset/resolvers.py

```diff
@@ -57,10 +57,10 @@ def get_file_upload_presigned_url(context, source, datasetUri: str = None, input
     return DatasetService.get_file_upload_presigned_url(uri=datasetUri, data=input)
 
 
-def list_owned_shared_datasets(context: Context, source, filter: dict = None):
+def list_all_user_datasets(context: Context, source, filter: dict = None):
     if not filter:
         filter = {'page': 1, 'pageSize': 5}
```

Comment:
This values are the default ones inside 
```
@staticmethod
 def paginated_dataset_shares(session, uri, data=None) -> [ShareObject]:
        query = ShareObjectRepository.query_dataset_shares(session, uri)
        return paginate(query=query, page=data.get('page', 1), page_size=data.get('pageSize', 5)).to_dict()
```

Maybe it's better to have default value as `{}` then?

Reply:
Same for ` list_owned_datasets`

Reply:
I agree on not setting it as default, but in this case the default is 10 not 5 `dataall.modules.datasets_base.db.dataset_repositories.DatasetRepository.paginated_all_user_datasets` I need to verify what is needed